### PR TITLE
Update installer to download to a central location

### LINKED ISSUE #4: Centralised download location
Suggestion/enhancement request: could the downloaded binary go in a central cache (i.e. in a dot-directory in the user's home directory) so that if you install the module in multiple projects or blow away your node_modules it doesn't have to download the binary again?

(Hoping we can get a similar change to nwjs-builder-phoenix as well.)

### PR DESCRIPTION
**Pull request by kierans**
I believe this should work @insightfuls

Fixes #4

### CODE REVIEW

File: installer/downloader.js [hunk 1]
```diff
@@ -72,8 +73,37 @@ module.exports = function(version, platform, arch) {
...
+  catch (e) {
+    fs.mkdirSync(dir);
+  }
+}
+
 function makeKey(version, platform, arch) {
   const suffix = platform === "win" ? ".exe" : "";
 
-  return version + "/chromedriver-" + platform + "-" + arch + suffix;
+  return path.join(version, "chromedriver-" + platform + "-" + arch + suffix);
```
Comment (insightfuls) [inline installer/downloader.js]: This change alters the key used to identify the artifact in S3 on Windows where the path separator will be different. Does that matter? Should you clear out any artifacts with the old keys and/or upload new ones?

File: installer/bin/chromedriver [hunk 2]
```diff
@@ -3,27 +3,26 @@
...
 
-const suffix = suffixForPlatform();
-const binPath = path.resolve(path.join(__dirname, "..", "chromedriver" + suffix));
-const args = process.argv.slice(2);
+const home = require("user-home");
 
-const cp = spawn(binPath, args);
-cp.stdout.pipe(process.stdout);
-cp.stderr.pipe(process.stderr);
+const utils = require(path.resolve(__dirname, "..", "utils.js"));
```
Comment (insightfuls) [inline installer/bin/chromedriver]: Why doesn't the simpler `require("../utils")` suffice here?

**Review by insightfuls — COMMENTED** [review]
An important question about the key used to identify artifacts in S3 on Windows.

File: installer/downloader.js (hunk 1 above)
Comment (kierans) [reply installer/downloader.js]: Good catch. I didn't think of that.

**Comment by kierans** [comment]
Bump @insightfuls

**Review by insightfuls — APPROVED** [review]
Looks good to me now. If you haven't already, please do test it before publishing.

**Pull request merged**